pkgs/applications/window-managers/awesome/default.nix

```diff
@@ -48,7 +48,10 @@ with luaPackages; stdenv.mkDerivation rec {
   LUA_PATH  = "${lgi}/share/lua/${lua.luaversion}/?.lua;;";
 
   postInstall = ''
-    wrapProgram $out/bin/awesome \
+    # Don't use wrapProgram or or the wrapper will duplicate the --search
```

Comment:
nitpick: `or or` -> `or`

Reply:
Seen this too late, sorry. But I fixed it with the changes in https://github.com/NixOS/nixpkgs/pull/61161